benchmark.py

```
@@ -407,13 +412,18 @@ async def run_search_mode(session: aiohttp.ClientSession, url: str, mode_name: s
     async def do_search(h: list[int], expected_match: bool) -> SearchResult:
         async with sem:
             start_time = time.time()
-            async with session.post(url, json={'query': h}) as response:
+            headers = {
+                'Content-Type': 'application/vnd.msgpack',
+                'Accept': 'application/vnd.msgpack'
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)